publish/canisters/BUILD.bazel

```diff
@@ -1,32 +1,25 @@
 load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
 load("//gitlab-ci/src/artifacts:upload.bzl", "upload_artifacts")
-load(
-    "//rs/nervous_system:compressed_wasm_size_limits.bzl",
-    "NNS_CANISTERS_MAX_SIZE_COMPRESSED_E5_BYTES",
-    "SNS_CANISTERS_MAX_SIZE_COMPRESSED_E5_BYTES",
-)
 
 # The list of canisters that are being published/uploaded to github as artifacts include
 # 1. Officially released canisters that can be downloaded/verified by 3rd parties - https://github.com/dfinity/ic/blob/master/.github/workflows/tag-release.yml
-# 2. Canisters required for dfx - https://github.com/dfinity/dfx-extensions/tree/main/extensions-utils/src/dependencies/download_wasms
-# 3. Mainnet canisters - https://github.com/dfinity/ic/blob/master/mainnet-canisters.bzl
+# 2. Mainnet canisters - https://github.com/dfinity/ic/blob/master/mainnet-canisters.bzl
+# 3. Canisters required for dfx - https://github.com/dfinity/dfx-extensions/tree/main/extensions-utils/src/dependencies/download_wasms
 CANISTERS = {
     "canister-creator-canister.wasm.gz": "//rs/rust_canisters/canister_creator:canister_creator_canister",
     "cycles-minting-canister.wasm.gz": "//rs/nns/cmc:cycles-minting-canister",
     "genesis-token-canister.wasm.gz": "//rs/nns/gtc:genesis-token-canister",
     "governance-canister.wasm.gz": "//rs/nns/governance:governance-canister",
-    "governance-canister_test.wasm.gz": "//rs/nns/governance:governance-canister-test",
```

Comment:
I think this may be used externally in testing, along with the test version of the sns-governance-canister

Reply:
Yes, it is. `dfx nns install` uses the test governance canister (thanks @dskloetd for confirming)

Reply:
Who may know if it is used or not ? Also what type of testing exactly ?

Reply:
Thanks @max-dfinity